src/app/pages/quickMaps/quickMaps.module.ts

```diff
@@ -34,6 +35,7 @@ import { BiomarkerComponent } from './pages/biomarkers/biomarker.component';
     RoutesModule,
     BaselineDetailsModule,
     ProjectionModule,
+    MatProgressSpinnerModule,
```

Comment:
Are you using this?
We have a spinner for a card and one for the whole page.
Where are we doing this spinning?

Reply:
added to the summary card component. Since it isn't actually a card like the other 2 cards with the graphs I didn't think it was possible to use the spinner for card